sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/Main.java

```diff
@@ -107,6 +168,14 @@ void runAll(OptionT options, NexmarkLauncher nexmarkLauncher) throws IOException
         saveSummary(options.getSummaryFilename(), configurations, actual, baseline, start, options);
         saveJavascript(options.getJavascriptFilename(), configurations, actual, baseline, start);
       }
+
+      executor.shutdown();
+      try {
+        if (!executor.awaitTermination(1L, TimeUnit.SECONDS)) {
+          throw new RuntimeException("Executor did not shutdown");
```

Comment:
nit: you're waiting for all runs to complete before reaching this point, and right after this you exit the program, so I'm not sure if it's worth waiting for executor termination

Reply:
deleted.